lib/pos_blockchain.go

```diff
@@ -114,28 +126,26 @@ func (bc *Blockchain) processBlockPoS(block *MsgDeSoBlock, currentView uint64, v
 
 // processOrphanBlockPoS validates that an orphan block is properly formed. If
 // an orphan block is properly formed, we will save it as Stored in the block index.
-// If not, we mark it as ValidateFailed.
+// As a spam-prevention measure, we will not store if a block fails the QC or leader check
+// and simply throw it away. If it fails the other integrity checks, we'll store it
+// as validate failed.
 func (bc *Blockchain) processOrphanBlockPoS(block *MsgDeSoBlock) error {
```

Comment:
Discussed offline. Marking as resolved since you'll have a followup PR for improvements elsewhere to reduce the number of cases where blocks will be stored in the DB